src/ads/components/ads-label.js

```diff
@@ -1,7 +1,8 @@
-vjs.AdsLabel = vjs.Component.extend({
+var vjsComponent = videojs.getComponent('Component');
+var AdsLabel = videojs.extend(vjsComponent, {
   /** @constructor */
-  init: function (player, options) {
-    vjs.Component.call(this, player, options);
+  constructor: function (player, options) {
+    vjsComponent.call(this, player, options);
 
```

Comment:
We could still use the `init` property instead of `constructor` because it is still supported even if it is deprecated (as described in the second exemple [here](https://github.com/videojs/video.js/wiki/5.0-Change-Details#videojs-50-subclassing-examples)).

**But** other parts of this PR will break with videojs 4.x anyway (like the `videojs.getComponent` method)... Is that okay if this plugin support only 5.x ? [An other plugin I need to use](https://github.com/videojs/videojs-contrib-dash) seems to support only the latest 5.x version, that's why I had to add support for 5.x in your plugin :wink: